modules/virt-linux-bridge-nad-port-isolation.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-linux-bridge-nad-port-isolation.adoc_{context}"]
+= Enabling port isolation for a Linux bridge NAD
+
+You can enable port isolation for a Linux bridge network attachment definition (NAD) so that virtual machines (VMs) or pods that run on the same virtual LAN (VLAN) can operate in isolation from one another. The Linux bridge NAD creates a virtual bridge, or _virtual switch_, between network interfaces and the physical network. You can then configure port numbers on the virtual bridge so that only specific traffic can reach each port. 
+
+Isolating ports in this way can provide enhanced security, efficient resource use, and performance improvements for VM workloads that run on the same node. 
```

Comment:
> efficient resource use, and performance improvements for VM workloads that run on the same node.

I am not aware using this feature improve reduce resources utilization or performance, could please share the reference for that?
If its not the case then I think this part should be omitted.

Reply:
Sorry, I took this from a source online. I removed those aspects of that statement leaving only "enhanced security" if this is OK?